src/bun.js/bindings/KeyObject.cpp

```diff
@@ -130,53 +130,119 @@ struct AsymmetricKeyValueWithDER {
 
 class KeyPassphrase {
 private:
-    WTF::CString pass_holder;
+    union {
+        WTF::CStringBuffer* strBuffer;
+        JSUint8Array* array;
+    } u;
+    uint8_t tag;
```

Comment:
I think this should be an enum with a default value to make it clear what each tag value is referring to, e.g.
```cpp
enum class Tag {
  None,
  StrBuffer,
  Array,
}
```
We could then add an initializer:
`Tag tag = Tag::None;`